lib/components/Label.vue

```diff
@@ -0,0 +1,119 @@
+<template>
```

Comment:
We have to rebuild this component. It contains a lot of code that is related to other components that creates tight coupling. This should just be a span/div that contains text. Or even better, a Label element directly.